ui/oopsyraidsy/oopsyraidsy.js

```diff
@@ -552,7 +552,7 @@ class MistakeCollector {
     // wipe then (to make post-wipe deaths more obvious), however this
     // requires making liveList be able to insert items in a sorted
     // manner instead of just being append only.
-    this.OnFullMistakeText('wipe', null, kPartyWipeText[this.options.Language || 'en']);
+    this.OnFullMistakeText('wipe', null, kPartyWipeText[this.options.ShortLocale || 'en']);
```

Comment:
For the same reason as the eureka ones above, I think this would be better served as "display language" rather than assuming that the user wants the system locale.  Can you undo this to use the parse language again?